### PR TITLE
feat: Add list of all available permissions

### LINKED ISSUE #86: Reference documentation for permissions exposed by the server
Ideally for both paper and spigot/craftbukkit

### PR DESCRIPTION
**Pull request by Leguan16**
Adds (i think) all permissions available with the default paper server.
My first time working with react so i already expect some improvements xd

closes #86.

### CODE REVIEW

**Comment by olijeffers0n** [comment]
Before I review, is gonna need a redirect as you changed a slug

**Comment by Leguan16** [comment]
> Before I review, is gonna need a redirect as you changed a slug

done

File: src/components/permission/PermissionSpec.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
+import { CommandData, PermissionData } from "@site/src/components/permission/Permissions";
+
+export const vanillaCommands: CommandData = {
+  "0": "minecraft.command",
+  "1": [
+    ["advancement", [], 2],
```
Comment (MiniDigger) [inline src/components/permission/PermissionSpec.tsx:6]: I am not really enjoying this data structure, can't we just use objects?

Comment (Leguan16) [reply src/components/permission/PermissionSpec.tsx:6]: It's the data structure used before in the vanilla command permissions thing. I just copied it.

Comment (MiniDigger) [reply src/components/permission/PermissionSpec.tsx:6]: see convo on discord, out of scope for this PR but if nobody else wants to I volunteer to migrate that into proper fields 
image

File: src/components/permission/PermissionSpec.tsx [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+    { name: "pardon-ip", aliases: [], opLevel: 3 },
+    { name: "particle", aliases: [], opLevel: 2 },
+    { name: "perf", aliases: [], opLevel: 4 },
+    { name: "place", aliases: [], opLevel: 2 },
+    { name: "playsound", aliases: [], opLevel: 2 },
+    { name: "publish", aliases: [], opLevel: 4 },
+    {
+      name: "random",
+      aliases: [],
+      opLevel: "Yes without sequence argument, No with sequence argument.",
```
Comment (MiniDigger) [inline src/components/permission/PermissionSpec.tsx:52]: Now that we have proper fields we don't need to reuse them for multiple purposes

File: src/components/permission/Permissions.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    return [];
+  }
+
+  return aliases
+    .map((x) => `/${x}`)
+    .map((x) => (<span className="no-wrap">{x}</span>) as AliasList)
+    .reduce((prev, curr) => [prev, <span>, </span>, curr]);
+}
+
+function getCommand(data: CommandData, x1: string) {
```
Comment (MiniDigger) [inline src/components/permission/Permissions.tsx:63]: x1 -> command

**Review by MiniDigger — APPROVED** [review]

**Review by olijeffers0n — CHANGES_REQUESTED** [review]
Prettier, Please

**Comment by Leguan16** [comment]
Not at home right now. Can do later.

**Review by olijeffers0n — APPROVED** [review]

**Pull request merged**